_includes/pbc/all/install-features/202304.0/marketplace/install-the-marketplace-inventory-management-feature.md

```diff
@@ -0,0 +1,774 @@
+            <parameter name="spy_product_offer_stock_all" column="*"/>
+        </behavior>
+    </table>
+</database>
+```
+
+Apply database changes and to generate entity and transfer changes:
+
+```bash
+console transfer:generate
+console propel:install
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied by checking your database:
+
+| DATABASE ENTITY | TYPE | EVENT |
+|-|-|-|
+| spy_merchant_stock | table | created |
+| spy_product_offer_stock | table | created |
+| spy_product_offer_availability_storage | table | created |
+
+{% endinfo_block %}
+
+### 3) Set up transfer objects
+
+Generate transfers:
+
+```bash
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes have been applied in transfer objects:
+
+| TRANSFER | TYPE | EVENT | PATH |
+|-|-|-|-|
+| MerchantStock | class | Created | src/Generated/Shared/Transfer/MerchantStockTransfer |
+| MerchantStockCriteria | class | Created | src/Generated/Shared/Transfer/MerchantStockCriteriaTransfer |
+| ProductAvailabilityCriteria | class | Created | src/Generated/Shared/Transfer/ProductAvailabilityCriteriaTransfer |
+| ProductConcreteAvailability | class | Created | src/Generated/Shared/Transfer/ProductConcreteAvailabilityTransfer |
+| ProductOfferAvailabilityRequest | class | Created | src/Generated/Shared/Transfer/ProductOfferAvailabilityRequestTransfer |
+| ProductOfferAvailabilityStorage | class | Created | src/Generated/Shared/Transfer/ProductOfferAvailabilityStorageTransfer |
+| ProductOfferStock | class | Created | src/Generated/Shared/Transfer/ProductOfferStockTransfer |
+| ProductOfferStockRequest | class | Created | src/Generated/Shared/Transfer/ProductOfferStockRequestTransfer |
+| ReservationRequest | class | Created | src/Generated/Shared/Transfer/ReservationRequestTransfer |
+| SpyMerchantStockEntity | class | Created | src/Generated/Shared/Transfer/SpyMerchantStockEntityTransfer |
+| SpyMerchantUserEntity | class | Created | src/Generated/Shared/Transfer/SpyMerchantUserEntityTransfer |
+| SpyProductOfferAvailabilityStorageEntity | class | Created | src/Generated/Shared/Transfer/SpyProductOfferAvailabilityStorageEntityTransfer |
+| SpyProductOfferStockEntity | class | Created | src/Generated/Shared/Transfer/SpyProductOfferStockEntityTransfer |
+
+{% endinfo_block %}
+
+### 4) Add Zed translations
+
+Generate a new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+### 5) Set up behavior
+
+Enable the following behaviors by registering the plugins:
+
+| PLUGIN | DESCRIPTION | PREREQUISITES | NAMESPACE |
+|-|-|-|-|
+| MerchantStockMerchantExpanderPlugin | Expands MerchantTransfer with related stocks. |  | Spryker\Zed\MerchantStock\Communication\Plugin\Merchant |
+| MerchantStockMerchantPostCreatePlugin | Creates default stock for the merchant. |  | Spryker\Zed\MerchantStock\Communication\Plugin\Merchant |
```

Comment:
* We have to add the following plugins to the list:
* \Spryker\Zed\MerchantStock\Communication\Plugin\AclMerchantPortal\MerchantStockAclEntityConfigurationExpanderPlugin
*